### PR TITLE
Notate throwing vs. non-throwing abstract operations

### LINKED ISSUE #371: Notate abstract ops that can throw
#366 was a good first step in making the flow of potential exceptions clearer in the spec. We should also consider notating whether an abstract operation can throw or not so that callers know whether to use perform or to handle any abrupt completions.

Here's an approach:
Sample screenshot hacked together with web inspector

### PR DESCRIPTION
**Pull request by domenic**
Fixes #371.

@tyoshino, pretty sure this is correct, but if you could do a quick check to make sure my throw/nothrow annotations are OK, that'd be helpful.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1833,7 +1844,7 @@ throughout the rest of this standard.
   1. Return *undefined*.
 </emu-alg>
 
-<h4 id="get-total-queue-size" aoid="GetTotalQueueSize">GetTotalQueueSize ( queue )</h4>
+<h4 id="get-total-queue-size" aoid="GetTotalQueueSize" throws>GetTotalQueueSize ( queue )</h4>
```
Comment (tyoshino) [inline index.bs]: nothrow?

File: index.bs [hunk 2]
```diff
@@ -1843,7 +1854,7 @@ throughout the rest of this standard.
   1. Return _totalSize_.
 </emu-alg>
 
-<h4 id="peek-queue-value" aoid="PeekQueueValue">PeekQueueValue ( queue )</h4>
+<h4 id="peek-queue-value" aoid="PeekQueueValue" throws>PeekQueueValue ( queue )</h4>
```
Comment (tyoshino) [inline index.bs]: nothrow?

**Comment by tyoshino** [comment]
Reviewed annotations!

**Comment by domenic** [comment]
Thanks for the careful review; both of your throw -> nothrow finds were correct!

**Pull request merged**